features/unit/abi_responsejsons/addMethod.json

```diff
@@ -0,0 +1 @@
+{"name":"add","args":[{"name":"","type":"uint32"},{"name":"","type":"uint32"}],"returns":{"type":"uint32"}}
```

Comment:
I think the args should be `[{"type":"uint32"},{"type":"uint32"}]`, i.e. an empty name should be dropped.

Also applies to other JSON files